src/libraries/System.Net.Http/tests/StressTests/HttpStress/Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN apt-get update -y && \
     apt-get upgrade -y && \
     apt-get install -y cmake clang ruby-dev gem lttng-tools libssl-dev && \
     gem install fpm
-RUN git clone --depth 1 --single-branch --branch v2.3.5 --recursive https://github.com/microsoft/msquic
+RUN git clone --depth 1 --single-branch --branch v2.4.4 --recursive https://github.com/microsoft/msquic
```

Comment:
do we need to build msquic? It feels like we should just consume it from packages.microsoft.com ... and simply get latest e.g. no need to vary about particular varian?

Reply:
note may be useful. It was not obvious to me. I'mwondering if `libsmquic-dbg` package would be useful to anybody besides us...